src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryViewLogic.java

```diff
@@ -1,12 +1,12 @@
-package com.vaadin.samples.bookstore.ui.inventory;
+package org.vaadin.example.bookstore.ui.inventory;
 
 import java.io.Serializable;
 
 import com.vaadin.flow.component.UI;
```

Comment:
Add comments to enter() about what is happening in the method

Reply:
init() too.

CancelProduct() should be moved downward so that the method order is logical

Reply:
Done.